operator/v1/types_ingress.go

```diff
@@ -200,6 +200,17 @@ type IngressControllerSpec struct {
 	// +optional
 	TuningOptions IngressControllerTuningOptions `json:"tuningOptions,omitempty"`
 
+	// bindOptions defines parameters for binding haproxy in ingress controller pods.
+	// All fields are optional and will use their respective defaults if not set.
+	// See specific bindOptions fields for more details.
+	//
+	//
+	// Setting fields within bindOptions is generally not recommended. The
+	// default values are suitable for most configurations.
+	//
+	// +optional
+	BindOptions IngressControllerBindOptions `json:"bindOptions,omitempty"`
```

Comment:
To omit when empty, it's better to use a pointer to `IngressControllerBindOptions`, otherwise `bindOptions: {}` would be marshaled.